source/bson-corpus/binary-vector-subtype.md

```diff
@@ -0,0 +1,94 @@
+# BSON Binary Subtype 9 - Vector
```

Comment:
This file doesn't belong in the bson corpus. Instead let's move it to its own folder like bson-decimal128 called `bson-vector/bson-vector.md`

Reply:
Yeah. It wasn't clear where to put it so I just put it somewhere.

Reply:
Changed.